fvm/src/syscalls/env.rs

```diff
@@ -0,0 +1,23 @@
+use super::Context;
+use crate::kernel::{Kernel, Result};
+
+pub fn timestamp(_context: Context<'_, impl Kernel>) -> Result<u64> {
```

Comment:
Just noting that we had discussed to put all this data behind a single struct, following the same patter as the `InvocationContext`, e.g. `ChainContext`. I'm happy to keep it as separate ops for now, it's always easy to add another syscall to retrieve all info at once for when the actor wants more than just a single field.

Reply:
IMO, we'd also want to move some fields out of the invocation context (epoch, network version, etc.) into the ChainContext (or EpochContext?).

(I'd prefer todo this _sooner_ rather than later, if possible)

Reply:
Sure, but I'd rather do this in a subsequent pr.